### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by knowthetech** [sw33tLie/bbscope#53]
can you add a feature to also fetch out of scope domains if we want?

**Comment by onlyvans404** [comment]
> can you add a feature to also fetch out of scope domains if we want?

yes like this *.example.com for sub.txt, api.example.com or example.com/api for api.txt and example.com for other.txt, add feature for remove .app and .exe too. sometimes the domain is reversed on h1, like this com.example

**Comment by sw33tLie** [comment]
Sorry for not replying earlier, but this is now supported :)
